timeseriestable/src/components/HistogramChart.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactElement, useMemo } from 'react';
+import { FormatOptions } from '@perses-dev/core';
+import { EChart, ModeOption, useChartsTheme } from '@perses-dev/components';
+import { use, EChartsCoreOption } from 'echarts/core';
+import { Box } from '@mui/material';
+import { BucketTuple } from '@perses-dev/prometheus/src/model';
+import { CustomChart } from 'echarts/charts';
+import { CustomSeriesRenderItemAPI, CustomSeriesRenderItemParams } from 'echarts';
+
+use([CustomChart]);
+
+export interface HistogramChartData {
+  buckets: BucketTuple[];
+}
+
+export interface HistogramChartProps {
+  width: number;
+  height: number;
+  data?: HistogramChartData;
+  format?: FormatOptions;
+  mode?: ModeOption;
+  exponential?: boolean;
+}
+
+export function HistogramChart({ width, height, data }: HistogramChartProps): ReactElement | null {
+  const chartsTheme = useChartsTheme();
+
+  const transformedData = useMemo(() => {
+    if (!data) return [];
+
+    return data.buckets.map((bucket) => {
```

Comment:
can you decompose the tuple in parameter here? like `[bucket, upperBound, lowerBound, count]` it will be more reasable then. 

Btw why we have string if we changed them to number afterward?